### ISSUE TITLE
No desktop notifications and desktop sound continuously dings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [max99x/pagemon-chrome-ext#255]
```
What steps will reproduce the problem? Please include a URL.

Any change in a url that you are monitoring.

What is the expected output? What do you see instead?

The url to pop up in the bottom corner with 1 ding.  Instead we are getting no 
pop up with it dinging continuously until you "mark as read"

What version of the Chrome are you using? On what operating system?
Version 28.0.1500.71 m  Windows 7

What are the error log messages? To view them, go to Wrench -> Tools ->
Extensions -> Developer Mode -> expand Page Monitor -> "background.htm" ->
Console.
Uncaught TypeError: object is not a function background.js:6
19
Uncaught TypeError: Object #<NotificationCenter> has no method 
'createHTMLNotification' background.js:2
3
Uncaught TypeError: Object #<NotificationCenter> has no method 'cr

Please provide any additional information below.
```

Original issue reported on code.google.com by `just...@gmail.com` on 11 Jul 2013 at 4:03

**Comment by GoogleCodeExporter** [comment]
```
This is happening to me too.  
My error is:
Uncaught TypeError: Object #<NotificationCenter> has no method 
'createHTMLNotification' background.js:2
I'm using the same OS and Chrome version.
```

Original comment by `CattCali...@gmail.com` on 11 Jul 2013 at 11:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `max99x` on 12 Jul 2013 at 3:01
* Changed state: **Accepted**
* Added labels: **Priority-Critical**, **Type-Defect**
* Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Issue 252 has been merged into this issue.
```

Original comment by `max99x` on 12 Jul 2013 at 3:02

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v3.3. Should roll out to all users over the next 24 hours.
```

Original comment by `max99x` on 12 Jul 2013 at 5:12
* Changed state: **Fixed**
